contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -253,6 +257,10 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
                 if (!areResultsPerMarketSet[gameId][typeId][playerId]) {
                     resultsPerMarket[gameId][typeId][playerId] = results;
                     areResultsPerMarketSet[gameId][typeId][playerId] = true;
+                    if (numOfTicketsToExercise > 0) {
+                        address[] memory activeTickets = manager.getActiveTicketsPerMarket(0, 20, gameId, typeId, playerId);
```

Comment:
instead of hardcoding the number to 20, maybe use numOfTicketsToExerciseOnGameResolution